src/azure-cli/azure/cli/command_modules/keyvault/_help.py

```diff
@@ -143,34 +163,171 @@
+    this operation does NOT return key material in a form that can be used outside the Vault or HSM
+    system, the returned key material is either protected to a HSM or to Vault itself. The intent
+    of this operation is to allow a client to GENERATE a key in one Vault or HSM instance, BACKUP the
+    key, and then RESTORE it into another Vault or HSM instance. The BACKUP operation may be used to
+    export, in protected form, any key type from Vault or HSM. Individual versions of a key cannot be backed
+    up. BACKUP / RESTORE can be performed within geographical boundaries only; meaning that a BACKUP from one
+    geographical area cannot be restored to another geographical area. For example, a backup
+    from the US geographical area cannot be restored in an EU geographical area. This operation
+    requires the key/backup permission.
+"""
+
+helps['keyvault key create'] = """
+type: command
+short-summary: Creates a new key, stores it, then returns key parameters and attributes to the client.
+long-summary: The create key operation can be used to create any key type in Vault or HSM. If the named
+    key already exists, Vault or HSM creates a new version of the key. It requires the keys/create permission.
+"""
+
 helps['keyvault key download'] = """
 type: command
 short-summary: Downloads the public part of a stored key.
 examples:
-  - name: Save the key with PEM encoding
+  - name: Save the key with PEM encoding.
     text: |
         az keyvault key download --vault-name MyKeyVault -n MyKey -e PEM -f mykey.pem
-  - name: Save the key with DER encoding
+  - name: Save the key with DER encoding.
     text: |
         az keyvault key download --vault-name MyKeyVault -n MyKey -e DER -f mykey.der
 """
 
+helps['keyvault key list'] = """
+type: command
+short-summary: Lists keys in the specified Vault or HSM.
+long-summary: Retrieves a list of the keys in the Vault or HSM as JSON Web Key structures that contain the
+    public part of a stored key. The LIST operation is applicable to all key types, however only the base
+    key identifier, attributes, and tags are provided in the response. Individual versions of a key are not
+    listed in the response. This operation requires the keys/list permission.
+"""
+
+helps['keyvault key list-deleted'] = """
+type: command
+short-summary: Lists the deleted keys in the specified Vault or HSM.
+long-summary: Retrieves a list of the keys in the Vault or HSM as JSON Web Key structures that contain the
+    public part of a deleted key. This operation includes deletion-specific information. The Get Deleted Keys
+    operation is applicable for vaults enabled for soft-delete. While the operation can be invoked on any
+    Vault or HSM, it will return an error if invoked on a non soft-delete enabled Vault or HSM. This operation
+    requires the keys/list permission.
+"""
+
+helps['keyvault key purge'] = """
+type: command
+short-summary: Permanently deletes the specified key.
```

Comment:
some short description missing `Vault or HSM`.

Reply:
@yungezz The original text doesn't contain `vault` or `key vault`, so I didn't modify the description.